src/main/java/core/basesyntax/strategy/impl/BirthdayDiscountService.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.strategy.impl;
+
+import core.basesyntax.strategy.DiscountService;
+
+public class BirthdayDiscountService implements DiscountService {
+    private double bdDiscount = 33;

```

Comment:
please use full words in naming
and if it is a constant field, let's make it look according to check style